app/views/edit_note.erb

```diff
@@ -0,0 +1,10 @@
+
+<div class="newnotebox">
+ <form action= "/note/<%=@note.id%>/edit" method= "post">
+  <input type="hidden" name="_method" value="put">
+
+
+  <p> Update Note
+    <input type = "text" name="description" value"<%=@note.description%>"></p>
+  <input type = "submit" value="update">
```

Comment:
input tags need to self close `<input type = "submit" value="update"/>` NOTE the ending `/>`